### PR TITLE
Portfolio: details popup window PR

### PR DESCRIPTION
**Pull request by normainobary2021**
In this PR I have made the following changes:
- Modified index.html to include popup screen elements for both mobile and desktop
- Modified style.css to include necessary styling for new popup screen functionality
- Modified main.js to include dynamic page rendering functionality for the popup window feature
- Added new background images for the hyperlink buttons.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -3,14 +3,282 @@ const close = document.getElementById('close');
...
+  const fullSpan3 = document.createElement('span');
+  const fullMdlP3 = document.createElement('p');
+  const fullSpan4 = document.createElement('span');
+  const fullSpan5 = document.createElement('span');
+  const fullSpan6 = document.createElement('span');
+  const seeLiveBtn = document.createElement('button');
+  const seeLiveI = document.createElement('i');
+  const seeSourceBtn = document.createElement('button');
+  const seeSourceI = document.createElement('i');
+
```
Comment (ishimwezachee) [inline main.js]: - [ ]  Your popup card should follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) such as  font, colours, images, text, margins.

Figma design             |  Your design
:-------------------------:|:-------------------------:
image  |  image

File: main.js [hunk 2]
```diff
@@ -3,14 +3,282 @@ const close = document.getElementById('close');
...
+  popupDiv.className = 'overlay';
```
Comment (ishimwezachee) [inline main.js:156]: - [ ]  Your button should have the size similar to the one from the Figma design
image
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @normainobary2021 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing 👍 
Project is running successfully 👍 
PR is well described 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -445,6 +443,145 @@ nav {
...
+}
+
+.wrapper .wrapper-btns i {
+  font-size: 12px;
+}
+
+.mdl-fullscreen {
+  display: none;
+}
+
```
Comment (leonardodiasb) [inline style.css]: - [ ] You did a great job fixing the position of your desktop popup but it still looks very off from the Figma template. Kindly try to follow the Figma design as much as possible. Your popup design has some issues like the background is a little bit transparent, the paddings of the desktop tech tags are no matching, the paddings of your mobile 'See Live'/'See Source' buttons are not matching, the image sizing in the mobile design is off, etc...
Your design:
Screen Shot 2022-07-22 at 4 24 53 PM
Screen Shot 2022-07-22 at 4 24 22 PM

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @normainobary2021 , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @normainobary2021, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by normainobary2021** [comment]
Thank you @Munsa1, @leonardodiasb and @ishimwezachee for your kind reviews. I appreciate every bit of it

**Pull request merged**